app/templates/base.html

```diff
@@ -20,19 +20,18 @@
       <div class="container">
         <div class="row">
           <div class="col-sm-2">
-            <a class="main-logo pull-left" href="#"></a>
+            <a class="main-logo pull-left" href="/"></a>
             <div class="navbutton">
               <a href="/apidocs/index.html">API</a>
               <a href="https://frictionlessdata.io/">About</a>
             </div>
           </div>
 
           <div class="col-sm-7 search-div">
-            <form class="form-inline">
+            <form class="form-inline" action="/search" method="GET">
               <div class="form-group">
-                <input type="text" class="form-control" id="searchbox" placeholder="Data Package">
+                <input type="text" class="form-control" id="searchbox" placeholder="Data Package" name="q">
```

Comment:
One comment: I feel that "Search ..." would be a better placeholder than "Data Package" - but that's something for later (add to your ux holding issue ;-) ...)